src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +56,46 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService
+                = (MovieSessionService) injector.getInstance(MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
                         fastAndFurious.getId(), LocalDate.now()));
+
+        AuthenticationService authenticationService
+                = (AuthenticationService) injector.getInstance(AuthenticationService.class);
+        User user;
+        try {
+            user = authenticationService.register("test@gmail.com", "1234");
+        } catch (Exception e) {
+            throw new RuntimeException("Can't register user", e);
+        }
+        try {
+            user = authenticationService.login("test@gmail.com", "1234");
+        } catch (AuthenticationException e) {
+            throw new RuntimeException("Can't login with that e-mail: " + user.getEmail(), e);
+        }
```

Comment:
Combine two blocks try-catch into one

Reply:
But mentors are usually told not to merge but to split them...

Reply:
In this case, the registration method does not throw any error at all, so we can combine these blocks or, in general, the registration method just make it out

Reply:
Let's try then. Thx.